src/scales/scale.logarithmic.js

```diff
@@ -179,62 +179,94 @@ module.exports = function(Chart) {
 		},
 		getPixelForValue: function(value) {
 			var me = this;
-			var start = me.start;
-			var newVal = +me.getRightValue(value);
-			var opts = me.options;
-			var tickOpts = opts.ticks;
-			var innerDimension, pixel, range;
+			var realValue = +me.getRightValue(value);
+			var reverse = me.options.ticks.reverse;
+			var lg = helpers.log10;
+			var exp = Math.floor(lg(me.minNotZero));
+			var significand = Math.floor(me.minNotZero / Math.pow(10, exp));
+			var firstTickValue = significand * Math.pow(10, exp);
+			var offset = 0;
+			var baseValue, innerDimension, pixel, range, sign;
 
+			if (reverse) {
+				range = {
```

Comment:
Can we avoid nesting `start` and `end` under `range`? (faster, lighter and easier to read)

```javascript
start = me.end;
end = me.start;
sign = -1;
```